common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java

```diff
@@ -219,19 +245,55 @@ private void clickPreviousPage() {
 
     @Override
     protected void onConfigUpdate(ConfigHolder configHolder) {
-        switch (configHolder.getFieldName()) {
-            case "buttonOnePage",
-                    "buttonFourPage",
-                    "buttonFivePage",
-                    "buttonSixPage",
-                    "buttonThreePage",
-                    "buttonTwoPage" -> {
-                if ((int) configHolder.getValue() < 1) {
-                    configHolder.setValue(1);
-                } else if ((int) configHolder.getValue() > MAX_BANK_PAGES) {
-                    configHolder.setValue(MAX_BANK_PAGES);
-                }
+        String valueString = (String) configHolder.getValue();
+
+        List<Integer> originalValues = parseStringToDestinations(valueString);
+
+        if (originalValues == null) {
+            configHolder.setValue(configHolder.getDefaultValue());
+            return;
+        }
+
+        int maxValue =
+                configHolder.getFieldName().equals("bankDestinations") ? MAX_BANK_PAGES : MAX_HOUSING_CONTAINER_PAGES;
+
+        List<Integer> newValues = modifyJumpValues(originalValues, maxValue);
+
+        if (!newValues.equals(originalValues)) {
+            String formattedConfig = newValues.stream().map(Object::toString).collect(Collectors.joining(","));
+
+            configHolder.setValue(formattedConfig);
+        }
+    }
+
+    private List<Integer> parseStringToDestinations(String destinationsStr) {
+        String[] destinationStrings = destinationsStr.split(",");
+
+        if (destinationStrings.length != 6) {
+            return null;
+        }
+
+        try {
+
+            return Arrays.stream(destinationStrings).map(Integer::parseInt).collect(Collectors.toList());
```

Comment:
For the sake of sanity, shouldn't you limit this to whatever slots we have?

Reply:
At that point I can just remove the check for it being 6 long right?

Reply:
Oh, I didn't see that up there. But yes, limiting is a better way than not parsing it at all.

Reply:
Actually, will it work if it's less thn 6?

Reply:
https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/stream/Stream.html#limit(long)

Reply:
How are those changes I've just made. If you don't enter 6 values it will add the missing values from the default